components/service/glean/README.md

```diff
@@ -56,11 +56,14 @@ class SampleApplication : Application() {
 
         // Initialize the Glean library. Ideally, this is the first thing that
         // must be done right after enabling logging.
-        Glean.initialize(applicationContext)
+        Glean.initialize(applicationContext, Configuration(applicationId = "glean"))
```

Comment:
Maybe to make the note below more obvious this should be `applicationId = "YOUR_APPLICATION_NAME_HERE"`?

Reply:
Good point!